01_fib/lexual/main_test.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+)
+
+func TestFibPositive(t *testing.T) {
+	numbers := []string{
+		"1",
+		"1",
+		"2",
+		"3",
+		"5",
+		"8",
+		"13",
+	}
+	runTest(t, 7, numbers)
+}
+
+func TestFibNegative(t *testing.T) {
+	numbers := []string{
+		"1",
+		"-1",
+		"2",
+		"-3",
+		"5",
+		"-8",
+		"13",
+	}
+	runTest(t, -7, numbers)
```

Comment:
Perhaps also have a test for edge case ie. fib(0)?

Reply:
Yeah can you please add a case for this as @jessodri suggested